shared/src/packages/sites/libs/validation-schemas/site-get-by-project.validation-schema.ts

```diff
@@ -1,12 +1,14 @@
 import joi from 'joi';
 
 import { type SiteGetByProjectParametersDto } from '../types/types.js';
+import { siteFilter } from './site-filter.validation-schema.js';
 
 const siteGetByProjectParameters = joi.object<
   SiteGetByProjectParametersDto,
   true
 >({
   projectId: joi.number().required(),
+  queryParameters: siteFilter,
```

Comment:
```suggestion\
```
it should be here. it validation for parameters

Reply:
you mean to do all validation in one file?  delete site-filter.validation-schema.ts and do yt all here?

Reply:
you should have 2 separate validation
1 for parameters
2 for query parameters